src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
If there are clashing members, I would alternatively just kill the problematic member in the old `japi`.

However, I'm missing how moving the annotation causes clashes? Isn't our annotation called `warmup` and not `Warmer`?